sklearn/feature_selection/base.py

```diff
@@ -120,3 +120,61 @@ def inverse_transform(self, X):
         Xt = np.zeros((X.shape[0], support.size), dtype=X.dtype)
         Xt[:, support] = X
         return Xt
+
+
+def wrapper_scorer(score_func, X, y=None):
+    """ A wrapper function around score functions. This function takes as
```

Comment:
The first line of a docstring should be a one sentence summary. See [PEP 257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings)